src/metabase/api/cache.clj

```diff
@@ -62,6 +62,19 @@
                                     "question"  :model/Card)
                                   :id [:in ids]))))
 
+(defn- check-cache-access [models id]
+  (if (or (nil? id)
+          ;; sometimes its a sequence and we're going to check for settings access anyway
+          (not (number? id))
+          (zero? id))
```

Comment:
What does a `zero?` ID signify?

Reply:
"root" strategy has `{:model "root" :model_id 0}`, and I imagine that's only for the administrators to edit.